srcpkgs/fiche/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'fiche'
+pkgname=fiche
+version=0.9.1
+revision=1
+build_style=gnu-makefile
+short_desc="Command line pastebin server for sharing terminal output"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://termbin.com"
+distfiles="https://github.com/solusipse/fiche/archive/$version.tar.gz"
+checksum=f0cb279a2c2a4c0d1debcf56785fd8731a1427d2524221678cf69c9aaa85e831
+
+CFLAGS="-lpthread"
+
+system_accounts="fiche"
+fiche_homedir="/var/tmp/fiche"
+make_dirs="/var/log/fiche 0750 fiche fiche
+ /var/tmp 1777 root root
```

Comment:
don't think creating /var/tmp is necessary

Reply:
Done